docs/modules/ROOT/pages/vshn-managed/postgresql/sla.adoc

```diff
@@ -21,6 +21,8 @@ metadata:
   namespace: <your-namespace>
 spec:
   parameters:
+    # instances must be either 2 or 3 when serviceLevel=guaranteed is set 
+    instances: 2
```

Comment:
I would do it with a callout, like the other explanations in this yaml file.

```suggestion
    instances: 2 <1>
```

and then put the explanation in the bottom:
<1> instances must be either 2 or 3 when serviceLevel=guaranteed is set 

You need to increment the other callouts though. And then it will render nice explanations for the code snippet.

Reply:
applied